Dockerfile_project

```diff
@@ -0,0 +1,5 @@
+FROM readysetgo:latest
+
+ENV GOPATH /go
+
+ADD . $GOPATH/src/github.com/heroku/silvia-runtime-university
```

Comment:
Some enhancements to consider for this:

* What about setting [`WORKDIR`](https://docs.docker.com/engine/reference/builder/#workdir) so when you `docker run .. bash` you get dropped into the project directory?
* Is there a useful default command you could set with [`CMD`](https://docs.docker.com/engine/reference/builder/#cmd) so `docker run -f Dockerfile_project` without arguments does something useful?